project-repo/review-process.md

```diff
@@ -52,6 +52,7 @@ The accepted `types` are:
 
 Don't hesitate to comment and reiterate! When you engage with a pull request, the expectation is that you remain responsive, and not let the pull request go stale.
 
+1. The submitter should strive to [sign their commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits). While this may be difficult to enforce for community contributions, we expect team members to follow this recommendation.
```

Comment:
I agree but it also implies that we trust our GPG keys, and so define a process to perform that.

Otherwise signing isn't just much more secure than using an email address.

Reply:
Agreed. I think though that this should be at the wider company level (on how to have a modern encryption key, which many of us should refresh btw)